.gitignore

```diff
@@ -21,3 +21,4 @@ crt_portal/NO_ZIP.txt
 *.mo
 # this is where we store uploaded attachments when USE_LOCALSTACK=False
 attachments/
+*.min.js*
```

Comment:
This is now too broad since we had some existing minified JavaScript that wasn't part of the build process (they were just manually stored in the repo):

- crt_portal/static/js/aria-autocomplete-1.2.3.min.js 
- crt_portal/static/js/uswds.min.js
- crt_portal/static/js/uswds.min.js.map

Reply:
Oh, the last two will just get rebuilt since we also copied the source `uswds.js` in, so this might only affect the `aria-autocomplete-1.2.3.min.js` script.

Reply:
I moved aria-autocomplete-1.2.3.min.js into another folder so it will not be built.  Everything else should build as we want.

Reply:
Cool, I like this approach. Is `precompiled` something you've seen as a folder name? Usually I see these named as `vendor` (e.g. [1](https://stackoverflow.com/questions/50052846/use-of-vendor-folder-in-web-design) and [2](https://tommcfarlin.com/the-vendor-directory/)). Happy to go with your recommendation but I wanted to suggest an alternative name that seems more common in my experience.

Reply:
vendor works for me.  I will update.

Reply:
updated